.ci/lint-plugin.sh

```diff
@@ -5,10 +5,13 @@ REPO_DIR=$(git rev-parse --show-toplevel)
 pushd "${REPO_DIR}" >/dev/null
 
 docker run \
-  -it \
   --rm \
   -v "$(pwd):/plugin" \
   buildkite/plugin-linter \
     --id lambdatest/lambdatest
 
-popd >/dev/null
\ No newline at end of file
+EXIT_CODE=$?
+
+popd >/dev/null
+
+exit $EXIT_CODE
```

Comment:
Why was this required?

Reply:
We are running the tests inside a docker container, which is run from this shell script. So the exit code we get from the tests(inside docker) are lost in the scope of the script. This makes sure that we catch that exit code and exit the shell script with the same exit code so that GitHub actions will know if the tests failed or not. 

If we don't do this, we will get the command line message that the tests failed but the actions will show that the checks have  passed (script ran successfully so exit code 0) even if the tests fail. Hope this answers the question.